.gitignore

```diff
@@ -23,9 +23,8 @@ docs/code-docs/build
 .jekyll-cache/
 .jekyll-metadata
 
-# Testing data
-tests/unit/saved_checkpoint/
-
-# Dev/IDE data
-.vscode
-.theia
+# scanning data
```

Comment:
@Queera - can you please explain the motivation behind these changes?  Mostly removing the other test data and IDEs?  It seems `*.so` is already listed above in this file on line 12.

what are */docs_site and scan.txt?